public_budget/models/budget_detail.py

```diff
@@ -34,7 +34,31 @@ class BudgetDetail(models.Model):
         related='budget_id.currency_id',
         readonly=True,
     )
+    amount = fields.Monetary(
+        compute='_compute_amount'
```

Comment:
please add a comma to the end of the line.
apply this to modifications field definition too